File: app-java/weathersdk/src/main/java/com/phunware/android/weathersdk/api/OpenWeatherAPI.java
package com.phunware.android.weathersdk.api;

import com.phunware.android.weathersdk.models.Weather;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface OpenWeatherAPI {
    @GET("data/2.5/weather")
    Call<Weather> getWeather(@Query("zip") String zipCode, @Query("units") String units);
}

File: app-kotlin/weathersdk/src/main/java/com/phunware/android/weathersdk/api/OpenWeatherAPI.kt
package com.phunware.android.weathersdk.api

import com.phunware.android.weathersdk.models.Weather
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface OpenWeatherAPI {

    @GET("data/2.5/weather")
    fun getWeather(@Query("zip") zipCode: String, @Query("units") units: String = "imperial"): Call<Weather>
}